main.go

```diff
@@ -219,6 +222,11 @@ func readPod(namespace string) (pod *kubectl.Pod) {
 }
 
 func readCloudSQLInstance(projectID string) (instance sqlproxy.CloudSQLInstance) {
+	// Allow to connect using only the instance connection name
+	// when user does not have `gcloud projects list` projec rights
```

Comment:
There is a typo in `projec`.

Reply:
you can make suggestions too :dancers: 

```suggestion
	// when user does not have `gcloud projects list` project rights
```